redfish-core/include/redfish.hpp

```diff
@@ -115,6 +115,7 @@ class RedfishService
 
         requestRoutesSystemLogServiceCollection(app);
         requestRoutesEventLogService(app);
+        requestRoutesCELogService(app);
```

Comment:
Should this be behind the if like the rest?

Reply:
This service applies for both DBus and Journal EventLog endpoints, so did not put it under ifdef. I think it is OK to leave it the way it is here?

If not, will add ifdef...

Reply:
Why not put these down below with the others? 
i.e. move to line 185

Reply:
My thinking was to keep these two EventLog/CELog service declarations together. Will go with your suggestion.
       requestRoutesEventLogService(app);
        requestRoutesCELogService(app);